resources/assets/js/stories/WordCounter/WordCounter.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+import { withKnobs, number, text } from "@storybook/addon-knobs";
+import { withIntl } from "storybook-addon-intl";
+import WordCounterWrapper from "../../components/WordCounter/WordCounterWrapper";
+import SkillsWordCounter from "../../components/ApplicantSkills/SkillsWordCounter";
+
+const stories = storiesOf("Components|Word Counter", module)
+  .addDecorator(withInfo)
+  .addDecorator(withKnobs)
+  .addDecorator(withIntl);
+
+stories.add(
+  "Word Counter",
```

Comment:
I added another story for WordCounterWrapper so you can adjust the word threshold with knobs.